src/Nox.Ui.Blazor.Lib/Components/NoxTypes/EditEntityExactlyOne.razor.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.AspNetCore.Components;
+using Nox.Types;
+using Nox.Ui.Blazor.Lib.Models;
+
+namespace Nox.Ui.Blazor.Lib.Components.NoxTypes;
```

Comment:
I would move this component out of NoxTypes, as it is just a dropdown not related to any Nox.Type